spring-boot/admin/src/main/resources/templates/admin/fragments/services/list.html

```diff
@@ -92,43 +87,37 @@ <h2 th:text="#{admin.services.list.title}">[Administration]</h2>
             </div>
         </div>
         <div th:include="admin/fragments/scripts :: scripts" th:remove="tag">Javascript Libraries</div>
-        <div th:include="admin/fragments/scripts :: initializeDataTables" th:remove="tag">Javascript Libraries</div>
+        <div th:include="admin/fragments/addOrderingToList :: addOrderingToServiceList" th:remove="tag">Javascript Libraries</div>
         <script>
             $(function () {
-
                 $("#infoDialog").dialog({
                     width: 'auto',
                     height: "auto",
                     autoOpen: false,
                     modal: true,
                     show: {
                         effect: "blind",
-                        duration: 1500
+                        duration: 1000
                     },
                     hide: {
                         effect: "blind",
                         duration: 1000
                     },
                     buttons: {
                         "Yes": function () {
-
                             var id = $(this).data('deleteItem');
-
                             var token = $("meta[name='_csrf']").attr("content");
                             var header = $("meta[name='_csrf_header']").attr("content");
 
-
                             $.ajax({
                                 url: '/admin/services/' + id + '.json',
-
                                 headers: {header: token},
                                 type: 'DELETE',
                                 success: function (result) {
                                     console.log('success:' + JSON.stringify(result));
-
                                     console.log('remove :' + id);
-
                                     $("#item" + id).remove();
+                                    location.reload();
```

Comment:
This probably leads to the problem that after adding a key and then deleting it, it is re-added. Maybe explicitly load the keys overview page instead of just reloading the current location (which could be "add")?

Reply:
Done as suggested.